examples/uaclient-status-expired.json

```diff
@@ -1,6 +1,12 @@
 {
     "version": "27.4.2~21.10.1",
     "effective": null,
+    "contract": {
+        "name": "UA Apps - Essential (Virtual)"
+    },
+    "account": {
+        "name": "user@domain.com"
+    },
```

Comment:
my quick play suggests that ua actually has more fields here. Is it simple for you to just regenerate these files?

Reply:
Yes, we can regenerate the files for sure. ua has way more fields indeed :)
I originally generated this file using the "canonical - staff" contract token and I don't have a clear vision if any of the fields contains sensitive information. There's a bunch of contract / account IDs in there that I did not want to risk disclosing.

Reply:
> I originally generated this file using the "canonical - staff" contract token and I don't have a clear vision if any of the fields contains sensitive information.

Ah yes that's a fair point, I don't know either!